node/src/components/consensus/protocols/highway.rs

```diff
@@ -20,7 +20,9 @@ use crate::{
     components::{
         chainspec_loader::Chainspec,
         consensus::{
-            consensus_protocol::{BlockContext, ConsensusProtocol, ConsensusProtocolResult},
+            consensus_protocol::{
+                BlockContext, ConsensusProtocol, ConsensusProtocolResult as CpResult,
```

Comment:
While changing `ConsensusProtocolResult` to `CpResult` makes this code more concise, I'm not sure it's part of this feature.  Perhaps we could put the changes in this file into another `NO TICKET` PR?

Reply:
Will do. :+1: 
Actually, maybe we should rename the type itself instead of just the import? And maybe it should be called `CpOutcome`, because a "result" I'd usually expect to be an alias for some `Result<_, _>`, or at least something that implements `Try`?